spec/integration/apo_spec.rb

```diff
@@ -6,18 +6,20 @@
   let(:user) { create(:user) }
   let(:new_apo_druid) { 'druid:zy987wv6543' }
   let(:new_collection_druid) { 'druid:zy333wv6543' }
-  let(:new_apo) { Dor::AdminPolicyObject.new(pid: new_apo_druid) }
-  let(:new_collection) { Dor::Collection.new(pid: new_collection_druid) }
 
   after do
     Dor::AdminPolicyObject.find(new_apo_druid).destroy # clean up after ourselves
     Dor::Collection.find(new_collection_druid).destroy # clean up after ourselves
   end
 
   before do
-    allow_any_instance_of(ApoForm).to receive(:register_model).and_return(new_apo)
-    allow_any_instance_of(CollectionForm).to receive(:register_model).and_return(new_collection)
-    allow(ApoController).to receive(:update_index).with(any_args)
+    # Inject pid into dor-services client's params so the destroys above work
```

Comment:
This sentence is missing a noun.

Reply:
I wonder if it's now possible to run this test without hardcoding pids.  Shouldn't dor-services-app and suri just give us one?